pkg/storage/driver/driver.go

```diff
@@ -29,6 +29,8 @@ var (
 	ErrReleaseNotFound = errors.New("release: not found")
 	// ErrReleaseExists indicates that a release already exists.
 	ErrReleaseExists = errors.New("release: already exists")
+	// ErrPackageTooLarge indicates that package too large,because of storage limit,reference: https://github.com/kubernetes/kubernetes/blob/release-1.18/staging/src/k8s.io/apiserver/pkg/server/config.go#L323
+	ErrPackageTooLarge = errors.New("Chart package too large,storage limit is 3145728")
```

Comment:
typo. Please add a space after the comma.

Also, the number 3145728 is not very descriptive. Taking this from a user's perspective: is this 3145728 bits? 3145728 bytes? 3145728 kilobytes?

I'd re-word this as "release: storage limit is 3MB" to remain consistent with the other error messages. The issue may not be related to the chart package being too large. You can, in theory, request for a package that's 3145727 bytes in size (just under the 3MB limit) and the rest of the secret metadata required brings it over the limit.

Reply:
done. move to PR:https://github.com/helm/helm/pull/8357. 
At the same time, I have an [issue](https://github.com/helm/helm/issues/8359) that has nothing to do with the PR.